MANUAL_REGRESSION.md

```diff
@@ -523,17 +523,17 @@ Given user opened the link with `?uploadFallback=false` flag
 2. Open the cross device link on mobile browsers with a malfunctioning camera or on mobile browsers that do not support getUserMedia (i.e. Safari on iOS10.3 or earlier, Chrome on iOS)
    - user won't see the "use the native camera mode instead" link
    - user should see `Unsupported browser` message
-   - user should see `Restart the process on Safari` message
+   - user should see `Restart the process on the latest version of Safari/Chrome` message
```

Comment:
LGTM, but I wonder if we need to be specific about what message the user will see depending on which mobile OS they're on since exact message displayed varies between Android/iOS - on iOS user will see `Restart the process on the latest version of Safari`, on Android it will be `Restart the process on the latest version of Google Chrome`

Reply:
I think they should be able to figure that out given that at the top we state (i.e. Safari on iOS10.3 or earlier, Chrome on iOS)